### PR TITLE
Install libsodium to all nodes

### PR DESCRIPTION
**Pull request by dincho**

### CODE REVIEW

File: ansible/setup.yml [hunk 1]
```diff
@@ -70,3 +72,26 @@
...
       tags:
         - dev-tools
+
+    - name: Download libsodium
+      unarchive:
+        src: https://github.com/jedisct1/libsodium/releases/download/{{ libsodium_version }}/libsodium-{{ libsodium_version }}.tar.gz
+        dest: /usr/local/src/
+        remote_src: yes
+      args:
+        creates: /usr/local/src/libsodium-{{ libsodium_version }}.tar.gz
```
Comment (lucafavatella) [inline ansible/setup.yml]: I am surprised unarchive creates a .tar.gz

File: ansible/setup.yml [hunk 2]
```diff
@@ -19,6 +19,8 @@
       - happi
       - UlfNorell
       - gorillainduction
+    libsodium_version: "1.0.16"
```
Comment (lucafavatella) [inline ansible/setup.yml]: I see https://github.com/aeternity/epoch/blob/66ea4da55bb919eac10985c53ac920b46fba6cc3/docker/Dockerfile-builder#L22-L28
Good

File: ansible/setup.yml (hunk 1 above)
Comment (lucafavatella) [reply ansible/setup.yml]: From the documentation at http://docs.ansible.com/ansible/latest/unarchive_module.html this is still unclear to me

Comment (dincho) [reply ansible/setup.yml]: This was a bug/obsolete configuration fixed in c14dfe6, the module is already idempotent (checks if dest: dir exists).

Comment (dincho) [reply ansible/setup.yml]: good catch btw

**Pull request merged**